Source/Core/Core/PowerPC/Jit64/Jit.cpp

```diff
@@ -236,7 +236,7 @@ void Jit64::WriteCallInterpreter(UGeckoInstruction inst)
 {
 	gpr.Flush();
 	fpr.Flush();
-	if (js.isLastInstruction)
+	if (js.op->opinfo->flags & FL_ENDBLOCK)
```

Comment:
Does this work with single stepping, when every instruction is at the end of the block?